modules/network-observability-resource-recommendations.adoc

```diff
@@ -0,0 +1,41 @@
+//
+// network_observability/configuring_operator.adoc
+
+:_content-type: REFERENCE
+[id="network-observability-resource-recommendations_{context}"]
+= Resource management and performance recommendations
+
+Network Observability requires a substantial amount of resources. There are settings to consider that can help you get an optimal setup without compromising on the observability.
```

Comment:
```suggestion
Network Observability can require a substantial amount of resources. There are settings to consider that can help you get an optimal setup without compromising on the observability.
```

Reply:
I can see that. Your suggestion is good with me.